tests/Aspire.Hosting.Tests/Milvus/AddMilvusTests.cs

```diff
@@ -89,7 +89,7 @@ public async Task MilvusCreatesConnectionString()
         var connectionStringResource = milvus.Resource as IResourceWithConnectionString;
 
         var connectionString = await connectionStringResource.GetConnectionStringAsync();
-        Assert.Equal($"Endpoint=http://localhost:19530;Key=pass", connectionString);
+        Assert.Equal($"Endpoint=http://localhost:19530;Key=root:pass", connectionString);
```

Comment:
These tests have all moved out of `Aspire.Hosting.Tests` and are in `Aspire.Hosting.Milvus.Tests` now. Can you sync main?

Reply:
doh, lots of changes, forgot to rebase...I will do that and resubmit I think. -- keep feedback coming and i'll address.